frontend/src/metabase/nav/components/AdminNavbar/AdminNavbar.tsx

```diff
@@ -30,7 +30,7 @@ export const AdminNavbar = ({
   adminPaths,
 }: AdminNavbarProps) => {
   return (
-    <AdminNavbarRoot className="Nav">
+    <AdminNavbarRoot className="Nav" data-testid="app-bar">
```

Comment:
can't we use `role` instead of this? 

seems [navigation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role) can be used